types/src/v17/wallet/into.rs

```diff
@@ -338,6 +338,11 @@ impl GetTransaction {
             self.block_index.map(|idx| crate::to_u32(idx, "block_index")).transpose()?;
 
         let txid = self.txid.parse::<Txid>().map_err(E::Txid)?;
+        let wallet_conflicts = self
+            .wallet_conflicts
+            .into_iter()
+            .map(|s| s.parse::<Txid>().map_err(E::Txid))
```

Comment:
We don't want to re-use the `Txid` error here. The error variants are name after the field not after the type. So we need a new variant `WalletConflicts`.

Reply:
New error variant added.